### PR TITLE
#1423 Release notes for 0.6.1

### PR DESCRIPTION
**Pull request by johannes-b**

### CODE REVIEW

File: releasenotes/releasenotes_V0.6.1.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
+# Release Notes 0.6.1
+
+This release introduces the new and shiny :sparkles: **Keptn's Bridge** :sparkles:
+
+* This new Keptn's Bridge is implemented on an AngularJS and NodeJS stack. An excerpt of its feature set shows:
```
Comment (christian-kreuzberger-dtx) [inline releasenotes/releasenotes_V0.6.1.md]: It's not AngularJS, just Angular :)

Comment (johannes-b) [reply releasenotes/releasenotes_V0.6.1.md]: Thanks. Yesterday, I forgot to ask.

File: releasenotes/releasenotes_V0.6.1.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+## Fixed Issues
+- Check K8s cluster version only for full installation [#1398](https://github.com/keptn/keptn/issues/1398)
+- Fixed memory leak in *mongodb-datastore* [#1440](https://github.com/keptn/keptn/issues/1440)
+- Use right mount path for persistant volume in *mongodb-datastore* [#1360](https://github.com/keptn/keptn/issues/1360)
+
+
+## Good to know / known limitations
+
+Addressed limitations:
+* ~~Distributors can not automatically reconnect to NATS cluster [#1209](https://github.com/keptn/keptn/issues/1209)~~
```
Comment (christian-kreuzberger-dtx) [inline releasenotes/releasenotes_V0.6.1.md]: This limitation is gone (addressed in #1209)

File: releasenotes/releasenotes_V0.6.1.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+* ~~Dynatrace SLI Service / Metrics API will change at the end of Q1 [#1282](https://github.com/keptn/keptn/issues/1282)~~
```
Comment (christian-kreuzberger-dtx) [inline releasenotes/releasenotes_V0.6.1.md]: we've updated the service, so this is also gone

File: releasenotes/releasenotes_V0.6.1.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+- Fixed memory leak in *mongodb-datastore* [#1440](https://github.com/keptn/keptn/issues/1440)
+- Use right mount path for persistant volume in *mongodb-datastore* [#1360](https://github.com/keptn/keptn/issues/1360)
+
+
+## Good to know / known limitations
+
+Addressed limitations:
+* ~~Distributors can not automatically reconnect to NATS cluster [#1209](https://github.com/keptn/keptn/issues/1209)~~
+* ~~Dynatrace SLI Service / Metrics API will change at the end of Q1 [#1282](https://github.com/keptn/keptn/issues/1282)~~ 
+* ~~The Quality-gates standalone version is currently not supported on OpenShift [#1197](https://github.com/keptn/keptn/issues/1197)~~
```
Comment (christian-kreuzberger-dtx) [inline releasenotes/releasenotes_V0.6.1.md]: OpenShift support is working now

**Comment by christian-kreuzberger-dtx** [comment]
One missing limitation/known issue is:

* Installation on AKS with K8s version before 1.15.5 might fail #1429

File: releasenotes/releasenotes_V0.6.1.md (hunk 2 above)
Comment (johannes-b) [reply releasenotes/releasenotes_V0.6.1.md]: Right, the above line 95 ("Addressed limitations") should say that the next three limitations are gone. 

When taking a look at the following section, is it still unclear? 
image

Comment (johannes-b) [reply releasenotes/releasenotes_V0.6.1.md]: Can you take a look at the current version?

File: releasenotes/releasenotes_V0.6.1.md (hunk 4 above)
Comment (johannes-b) [reply releasenotes/releasenotes_V0.6.1.md]: Please take a look at the updated version.

File: releasenotes/releasenotes_V0.6.1.md (hunk 3 above)
Comment (johannes-b) [reply releasenotes/releasenotes_V0.6.1.md]: Please take a look at the updated version.

**Pull request merged**